app/controllers/editor/top_fixed_posts_controller.rb

```diff
@@ -0,0 +1,63 @@
+class Editor::TopFixedPostsController < Editor::ApplicationController
+  def index
+    @top_fixed_posts = top_fixed_posts.includes(:post).ordered.page(params[:page]).per(200)
+  end
+
+  def new
+    @top_fixed_post = top_fixed_posts.build
+    @candidate_posts = candidate_posts
+  end
+
+  def create
+    @top_fixed_post = top_fixed_posts.build(top_fixed_post_params)
+
+    if @top_fixed_post.save
+      redirect_to [:editor, :top_fixed_posts], notice: "トップ固定記事の設定が作成されました"
+    else
+      @candidate_posts = candidate_posts
+
+      render :new
+    end
+  end
+
+  def edit
+    @top_fixed_post = top_fixed_posts.find(params[:id])
+    @candidate_posts = candidate_posts
+  end
+
+  def update
+    @top_fixed_post = top_fixed_posts.find(params[:id])
+
+    if @top_fixed_post.update(top_fixed_post_params)
+      redirect_to [:editor, :top_fixed_posts], notice: "トップ固定記事の設定が更新されました"
+    else
+      @candidate_posts = candidate_posts
+
+      render :edit
+    end
+  end
+
+  def destroy
+    top_fixed_post = top_fixed_posts.find(params[:id])
+    top_fixed_post.destroy
```

Comment:
Why don't you handle failing?

destroy will return false if we can't destroy it.

http://api.rubyonrails.org/v4.2.7.1/classes/ActiveRecord/Persistence.html#method-i-destroy